### PR TITLE
COMMERCE-6975 (1/2)

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java [hunk 1]
```diff
@@ -62,6 +62,37 @@ public CommerceOrder addCommerceOrder(
 			userId, groupId, commerceAccountId, commerceCurrencyId);
 	}
 
+	@Override
+	public CommerceOrder addCommerceOrder(
+			long userId, long groupId, long commerceAccountId,
+			long commerceCurrencyId, long commerceOrderTypeId)
```
Comment (dsitu) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:68]: Remove `long userId`, remote services must not allow anyone to pass in `userId`. We should use `getUserId()`.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by liulukiwi** [comment]
ci:test:bundle

**Comment by liulukiwi** [comment]
I run the bundle for [LRQA-69350](https://issues.liferay.com/browse/LRQA-69350) use only, sorry for the bother.

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**